lumicks/pylake/force_calibration/power_spectrum.py

```diff
@@ -12,7 +12,7 @@ class PowerSpectrum:
     frequency : numpy.ndarray
         Frequency values for the power spectrum. [Hz]
     power : numpy.ndarray
-        Power values for the power spectrum (typically in V^2/s).
+        Power values for the power spectrum (typically in 0.5 V^2/s).
```

Comment:
This one was wrong in a different way before:
```suggestion
        Power values for the power spectrum (typically in 0.5 V^2/Hz).
```

Reply:
> What are the tradeoffs of adding `0.5 *` to the label vs. multiplying the result by 2? From my point of view, a unit with a non-SI scaling factor is a bit weird. But that could just be what I'm used to and a different convention makes sense here. I'm just wondering what the reasoning is.

Nah, I agree with you. I see three options:

1. We update `.power` to use the more common SI convention. But this is a breaking change. It's true that this would be a release with many of those already, but what I don't like is that it has the risk of passing in user code silently. If we were to start over, this would have my preference, but unfortunately, this wasn't caught before the first release. Note that this would also mean updating a number of equations; which would then be different by a factor two from the paper. This might confuse people looking at the code, but a comment could easily mitigate that part.
If we go this route, I would maybe defer this PR to until we are ready to begin merging the active calibration stuff (because otherwise I see a lot of room for potential mistakes with factors of two, since it uses the current convention).

2. We keep the convention of the first force calibration paper and update the axis labels to correctly reflect what's in them (that's what is currently in this PR).

3. We change what we plot and multiply things by two prior to plotting. This in my opinion is the worst option, because it makes users expect that the `power` property will be what we plot.

I was thinking of doing 2 now, and 1 once we bring the two calibration scripts together again. What do you think?

Reply:
> This one was wrong in a different way before:

Right. Fixing.

Reply:
Definitely not 3.

I'm not sure that I understand the reasoning to do both 2 and 1. I'd be fine with just 2 if that's the prevailing convention in the field. I would not want to complicate this if it also has an impact on active calibration and we'll always need to worry about the convention.

Reply:
I wouldn't go as far as to call it a field-wide definition. It's used by 5 out of the 6 papers that I have. The active calibration paper (most recent) actually has the "correct factor" of two, but the implementation we have switches those equations to the old convention (that we currently have in Pylake). Note that the 5 that use the convention we use always just put `arbitrary units` on their axis.

Functions in `scipy` for instance, all use the "correct" definition (not ours).

Unfortunately, because of the first paper using an unconventional form to begin with, we'll always have to be careful about it.

Long term, my preference would be to express it in `V^2/Hz` but that would mean breaking backward compatibility and redefining `PowerSpectrum.power`. The functionality is still flagged as beta now, but it still doesn't feel right to change the definition of an existing attribute on people. We could think of a new name for `power`, but quite frankly, that's the best name for it. I guess we could deprecate `power` and add `power_density` but that feels overlong.

In either case, what we have on the axis now is incorrect. Maybe we can briefly discuss tomorrow post-standup.

Reply:
If that's the case and the current `0.5 V^2/Hz` convention is the uncommon one, I'd pull the bandaid off right now rather than wait to do it in the future. The feature is clearly marked as alpha in the docs and I can't imagine many people are using it right now (as opposed to just using the calibration from Bluelake). Going for the breaking change now would be as painless as it will ever be.

And since we are working on pulling this into Bluelake (not yet live), I'd rather get the correct scale from the start. We do also have to consider that this will result in a 2x PSD scale change in the force calibration window when we switch from LDAT to pylake but AFAIK the only thing that people rely on there is the shape, now the exact values.

Reply:
Yeah, it'll just be a vertical shift on the graph. I indeed checked internally and people only rely on the shape.

Alright. Pulling the bandaid off it is. We will really have to be extra careful pulling in the active calibration materials too, since they still operate in `0.5 V^2/Hz`.